rocketpy/Function.py

```diff
@@ -1676,27 +1713,200 @@ def __interpolateAkima__(self):
                              2*yl + 2*yr)/(xl-xr)**3"""
         self.__akimaCoefficients__ = coeffs
 
+    def __neg__(self):
+        """Negates the Function objetive. The result has the same effect as
+        multiplying the Function by -1.
+
+        Returns
+        -------
+        Function
+            The negated Function object.
+        """
+        if isinstance(self.source, np.ndarray):
+            neg_source = np.column_stack((self.xArray, -self.yArray))
+            return Function(
+                neg_source,
+                self.__inputs__,
+                self.__outputs__,
+                self.__interpolation__,
+                self.__extrapolation__,
+            )
+        else:
+            return Function(
+                lambda x: -self.source(x),
+                self.__inputs__,
+                self.__outputs__,
+                self.__interpolation__,
+                self.__extrapolation__,
+            )
+
+    def __ge__(self, other):
+        """Greater than or equal to comparison operator. It can be used to
+        compare a Function object with a scalar or another Function object.
+        This has the same effect as comparing numpy arrays.
+
+        Note that it only works for Functions if at least one of them is
+        defined by a set of points so that the bounds of the domain can be
+        set.
+        If both are defined by a set of points, they must have the same
+        discretization.
+
+        Parameters
+        ----------
+        other : scalar or Function
+
+        Returns
+        -------
+        numpy.ndarray of bool
+            The result of the comparison one by one.
+        """
+        otherIsFunction = isinstance(other, Function)
+
+        if isinstance(self.source, np.ndarray):
+            if otherIsFunction:
+                try:
+                    return self.yArray >= other.yArray
```

Comment:
Since we are not checking if `self.xArray` and `other.xArray` are the same, it seems that if `len(self.xArray) == len(other.yArray)`, then an erroneous comparison will be made without raising any issues.

Is this behavior expected?

This extends to the remaining comparison methods.

Reply:
I admit that this behavior stretches quite a bit the intuitive side of a comparison, but I don't think that raising an error is necessary here. Even if the domains are different, the `yData` is there to be compared.